Chummer/data/weapons.xml

```diff
@@ -53,6 +53,7 @@
     <category blackmarket="Weapons" type="energy" gunneryspec="Energy">Laser Weapons</category>
     <category blackmarket="Weapons" type="glauncher" gunneryspec="Artillery">Grenade Launchers</category>
     <category blackmarket="Weapons" type="mlauncher" gunneryspec="Rocket">Missile Launchers</category>
+    <category blackmarket="Weapons" type="firefighting cannons">Firefighting Cannons</category>
```

Comment:
Gunnery spec?

Reply:
Any idea which would be appropriate?

Reply:
Note that you don't need this weapon category if you set `noextra="True"` for Ammo: Water and Ammo: Fire Extinguishant.